audio/src/fetch.rs

```diff
@@ -61,7 +61,6 @@ impl AudioFileOpenStreaming {
         });
 
         let mut write_file = NamedTempFile::new().unwrap();
-        write_file.set_len(size as u64).unwrap();
```

Comment:
My apologies - I meant to ask for clarification on this.

This call pre-allocates the file rather than dynamically growing it as the stream produces data. This seems reasonable and potentially more performant. Do you know if this was done for any other reason? I'm not clear on whether this call has a meaningful impact after fs caching and would like to at least add a comment here to clarify the desired outcome.

Reply:
Personally I don't think changing any behaviour here is a good idea. If it's worth doing then a dedicated PR with some perf data would be better, IMO.

Reply:
>Do you know if this was done for any other reason?

I'd reckon this is in place as a "better safe than sorry" measure. I don't see any detrimental effects of it right of the bat.  Eitehrway, don't think we should change it as part of this dependency upgrade commit/PR.

Reply:
You can manually change the checksums in https://github.com/librespot-org/librespot/blob/master/protocol/files.rs (I'd just change the last digit of each).

I think that's the easiest way to trigger the rebuild.